README.md

```diff
@@ -51,8 +51,6 @@ See the [format description](FORMAT.md).
 
 ## TODO
 
-  * Set returncode on check/diff: 0 (no changes), 1 (changes), 2
-    (suspicious changes), negative (failure); or something similar
```

Comment:
I'd like this to be documented somewhere in the help message for the application. I'm hoping we can add this via the [after_help](https://docs.rs/clap/2.26.2/clap/struct.App.html#method.after_help) method. Probably want this to look something like:

```
RETURN CODE:
    0       Success
    1       Changes
    2       Suspicious changes
   -1       Error
```

And we probably want this under the respective subcommand.